src/features/Header/Header.jsx

```diff
@@ -0,0 +1,102 @@
+/* eslint-disable jsx-a11y/no-static-element-interactions */
+/* eslint-disable jsx-a11y/click-events-have-key-events */
+/* eslint-disable react/jsx-no-bind */
+/* eslint-disable react/button-has-type */
+import React, { useEffect, useState } from 'react'
+import Modal from 'react-modal'
+import styled from './Header.module.scss'
+import wallpaper from '../../../public/img/header-background.png'
+// eslint-disable-next-line import/extensions
+import AddMovie from '../../base/AddMovie/index.jsx'
+// eslint-disable-next-line import/extensions
+import Search from '../../base/Search/index.jsx'
+// eslint-disable-next-line import/extensions
+import SearchButton from '../../base/SearchButton/index.jsx'
+// eslint-disable-next-line import/extensions
+import ErrorBoundary from '../ErrorBoundary/ErrorBoundary.jsx'
+// eslint-disable-next-line import/extensions
+// eslint-disable-next-line no-unused-vars
+import AddMovieModal from '../AddMovieModal/AddMovieModal.jsx'
+
+function Header() {
+  // eslint-disable-next-line no-unused-vars
+  const [input, setInput] = useState('')
+  const [toggle, setToggle] = useState(false)
+  const [isOpen, setIsOpen] = useState(false)
+
+  useEffect(() => {
+    if (toggle) {
+      document.write(Error('Crashed with Error Boundary!'))
+      throw new Error('Crashed with Error Boundary!')
+    }
+  }, [toggle])
+
+  function toggleModal() {
+    setIsOpen(!isOpen)
+  }
+
+  return (
+    <nav
+      style={{ backgroundImage: `url(${wallpaper})` }}
+      className={`${styled.parent} navbar navbar-light p-5`}
+    >
+      <div className={`container-fluid ${styled.containerHeader}`}>
```

Comment:
to combine classes a bit more comfortably, install cpm package clsx. It'll allow you to make this line into:
```suggestion
      <div className={clsx('container-fluid', styled.containerHeader)}>
```

Reply:
Ok, It was so multi-writing.
 thank you for letting me know this package)

Reply:
done